src/components/menu/index.js

```diff
@@ -323,6 +323,14 @@ const Menu = () => {
                             {t('Players')}
                         </Link>
                     </li>
+                    <li className="submenu-wrapper submenu-items overflow-member" key="menu-converter" data-targetid="converter">
+                        <Link
+                            to="/converter"
+                            //onClick={setIsOpen.bind(this, false)}
+                        >
+                            {t('Converter')}
```

Comment:
Thanks for the PR! I think the title in the menu should be "Currency Converter" Since just by looking at the menu title, it's currently not very clear what "Converter" means.

Reply:
You're right